app/glossary.md

```diff
@@ -247,6 +251,10 @@ Used to predict and manage the ITT market.
 
 Outstanding schools that work with others to provide high quality training to new and experienced staff. Cannot be lead schools or providers.
 
+### Teaching Vacancies (TVS)
+
+Service to [find and apply for teaching jobs](https://teaching-vacancies.service.gov.uk/).
```

Comment:
```suggestion
A service for [advertising, finding and applying for teaching jobs](https://teaching-vacancies.service.gov.uk/). Capitalised. Do not use the acronym publicly.
```

Reply:
Should `vacancies` be capitalised?

Reply:
Possibly not, but it is on [the service](https://teaching-vacancies.service.gov.uk/).